index.html

```diff
@@ -72,5 +72,166 @@ <h1 id="primary">Hey There. <br />I'm Lee</h1>
         </li>
       </ul>
     </section>
+    <section id="section-bac">
+      <div id="heading-second">
+        <h1 id="heads">My Recent Works</h1>
+      </div>
+      <section id="sec">
+        <div class="card" id="card">
+          <img src="./assets/first-card.png" alt="card image" />
+          <div class="description">
+            <p id="head1">Multi-Post Stories</p>
+            <p id="contents1">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <div class="buttons">
+              <button class="button1 butcss">css</button>
+              <button class="button1 buthtm">html</button>
+              <button class="button1 butboot">bootstrap</button>
+              <button class="button1 butrub">ruby</button>
+            </div>
```

Comment:
- [ ] [optional] Kindly ensure that you wrap the projects technologies with `ul` tag not `div`, each technology in a list `li` tag. `ul `tag is semantics but `div `tag has no meaning. 👍 Kindly apply this to all the projects cards. 😄 .